src/jdk.jlink/share/classes/jdk/tools/jlink/internal/LinkableRuntimeImage.java

```diff
@@ -62,7 +68,38 @@ public static boolean isLinkableRuntime() {
 
     private static InputStream getDiffInputStream(String module) throws IOException {
         String resourceName = String.format(DIFF_PATTERN, module);
-        return LinkableRuntimeImage.class.getModule().getResourceAsStream(resourceName);
+        return JDK_JLINK_MOD.getResourceAsStream(resourceName);
```

Comment:
FYI you can use LinkableRuntimeImage.class.getResourceAsStream here as the resource is in the current module.

Reply:
It's very odd, but when I attempt this then the resource is not found. It seems to fail on the module name verification. For example: `jlink --help | tail -n2` shows as `disabled` for an enabled linkable runtime image.

Reply:
Without reproducing, I can't immediately say why Class.getResourceAsStream would fail here. This is a good API for locating resources in the caller's module.